site/tutorials/gettings_started.md

```diff
@@ -0,0 +1,293 @@
+
+```json
+[
+  {"a": "C", "b": 2},
+  {"a": "C", "b": 7},
+  {"a": "C", "b": 4},
+  {"a": "D", "b": 1},
+  {"a": "D", "b": 2},
+  {"a": "D", "b": 6},
+  {"a": "E", "b": 8},
+  {"a": "E", "b": 4},
+  {"a": "E", "b": 7}
+]
+```
+
+To visualize this data with Vega-Lite, we can add it directly to a Vega-Lite specification (we just formatted the array to make it more compact).
+
+```json
+{
+  "data": {
+    "values": [
+      {"a": "C", "b": 2}, {"a": "C", "b": 7}, {"a": "C", "b": 4},
+      {"a": "D", "b": 1}, {"a": "D", "b": 2}, {"a": "D", "b": 6},
+      {"a": "E", "b": 8}, {"a": "E", "b": 4}, {"a": "E", "b": 7}
+    ]
+  }
+}
+```
+
+The `data` property defines the data source of the visualization. For now, we will directly set the `values` property but later we will so how we can use other data sources.
+
+## Visualize Data with Marks
+
+This specification only adds a data source but we haven't defined yet how the the data should be visualized. By setting the `mark` property to `point`, we tell Vega-Lite to render point marks.
+
+<div class="vl-example">
+{
+  "data": {
+    "values": [
+      {"a": "C", "b": 2}, {"a": "C", "b": 7}, {"a": "C", "b": 4},
+      {"a": "D", "b": 1}, {"a": "D", "b": 2}, {"a": "D", "b": 6},
+      {"a": "E", "b": 8}, {"a": "E", "b": 4}, {"a": "E", "b": 7}
+    ]
+  },
+  "mark": "point"
+}
+</div>
+
+Now, Vega-Lite renders a point. In fact, in renders one point for each of the 9 objects in the array.
+
+## Encode Properties of the Data as Visual Properties
+
+Seeing all points on top of each other is not particularly useful so to visually separate the points, we can *encode* the property `a` of the data as the x-location of the points.
+
+<div class="vl-example">
+{
+  "data": {
+    "values": [
+      {"a": "C", "b": 2}, {"a": "C", "b": 7}, {"a": "C", "b": 4},
+      {"a": "D", "b": 1}, {"a": "D", "b": 2}, {"a": "D", "b": 6},
+      {"a": "E", "b": 8}, {"a": "E", "b": 4}, {"a": "E", "b": 7}
+    ]
+  },
+  "mark": "point",
+  "encoding": {
+    "x": {"field": "a", "type": "ordinal"}
+  }
+}
+</div>
+
+Here, we added an encoding to our specification.
+
+```json
+...
+"encoding": {
+  "x": {"field": "a", "type": "ordinal"}
+}
+...
+```
+
+this encoding maps the values for field `a` to the *encoding channel* `x` (the x-location of the points). In addition to the mapping, we also have to specify the data type. Vega-Lite currently supports four data types: `ordinal`, `nominal`, `quanititative`, and `temporal`. The difference between these types is explained in the [documentation for data types]({{stite.baseurl}}/docs/encoding.html#types). It is required that you provide a data type for an encoding.
+
+In the visualization that we just crated, Vega-Lite automatically added an axis with labels for the different categories as well as an axis title. However, 3 points are still overlapping in each category. So far, we have only defined a visual encoding for the field `a`. We can add another visual encoding for the field `b`.
```

Comment:
> crated
created

> 3 points are still overlapping in each category
3 points in each category are still overlapping.

> We can add another visual encoding for the field `b`.
We can also map `b` to `y` channel.